### PR TITLE
ticket 5

### PR DESCRIPTION
**Pull request by CavBridge**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -92,4 +88,85 @@ describe("ARTICLES", () => {
...
+          expect(body.article).toEqual({
+            article_id: article_id,
+            title: "Student SUES Mitch!",
+            topic: "mitch",
+            author: "rogersop",
+            body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+            created_at: "2020-05-06T01:14:00.000Z",
+            votes: 0,
+          });
+        });
```
Comment (scar1377) [inline __tests__/app.test.js:148]: This should be a 400 because it's missing the required field inc_votes

File: models/articles.js [hunk 2]
```diff
@@ -11,3 +11,17 @@ exports.fetchArticlesById = (article_id) => {
       return rows;
     });
 };
+
+exports.patchArticleVotes = (article_id, newVotes = 0) => {
```
Comment (scar1377) [inline models/articles.js:15]: when send {} newVotes will be undefined, then there should be an error rather than default to 0.

**Review by scar1377 — CHANGES_REQUESTED** [review]
Hi Cavner, you're very close, just need to deal with {} as an error 400 -missing required fields

File: __tests__/app.test.js (hunk 1 above)
Comment (CavBridge) [reply __tests__/app.test.js:148]: Hi Scarlett I was with Stephen and he told me to change the 404 to a 200

Comment (CavBridge) [reply __tests__/app.test.js:148]: *400

**Pull request merged**